### DISCUSSION TITLE
How to inspect VSelect's menu HTML in devtools?

### DISCUSSION THREAD

**Discussion by binaryfunt** [vuetifyjs/vuetify#16880] [Q&A] [answered]
Every time I try to inspect the HTML of a `v-select`'s menu, the menu closes as soon as I click something in the devtools panel. I have tried manually triggering `$0.click()` from the console on various elements that make up the `v-select`, but to no avail.

Vuetify 3.1.7

**Comment by KaelWD** [comment, marked as answer]
#16860

**Reply by binaryfunt** [reply]
Thanks! (Answer is (Chromium) devtools > Rendering > "Emulate a focused page")
